.github/workflows/test.yml

```diff
@@ -291,17 +291,22 @@ jobs:
         name: Packages
         path: dist
 
-    - name: Set up Python ${{ matrix.python }}
-      uses: actions/setup-python@v5
+    # - name: Set up Python ${{ matrix.python }}
+    #   uses: actions/setup-python@v5
+    #   with:
+    #     python-version: ${{ matrix.python }}
+    #     check-latest: true
+    #     allow-prereleases: true
+
+    - name: Install the latest version of uv
+      uses: astral-sh/setup-uv@v6
```

Comment:
I think there's another job somewhere already doing this. Pytest shouldn't stop testing againsts normal python builds.